tests/integration/golang/mlflow/run/create_test.go

```diff
@@ -128,27 +236,62 @@ func (s *CreateRunTestSuite) Test_Error() {
 			),
 		},
 		{
-			name: "CreateRunWithNotExistingExperiment",
+			name:      "CreateRunWithNotExistingNamespaceAndExistingExperimentID",
+			namespace: "not_existing_namespace",
 			request: request.CreateRunRequest{
-				ExperimentID: "1",
+				ExperimentID: fmt.Sprintf("%d", *experiment.ID),
 			},
 			error: api.NewResourceDoesNotExistError(
-				`unable to find experiment with id '1': error getting experiment by id: 1: record not found`,
+				`unable to find namespace with code: not_existing_namespace`,
+			),
+		},
+		{
+			name:      "CreateRunWithNotExistingExperiment",
+			namespace: "default",
+			request: request.CreateRunRequest{
+				ExperimentID: fmt.Sprintf("%d", nonExistingExperimentID),
+			},
+			error: api.NewResourceDoesNotExistError(
+				fmt.Sprintf(
+					`unable to find experiment with id '%d': error getting experiment by id: %d: record not found`,
+					nonExistingExperimentID,
+					nonExistingExperimentID,
+				),
+			),
+		},
+		{
+			name:      "CreateRunWithExistingNamespaceAndNotExistingExperiment",
+			namespace: "default",
+			request: request.CreateRunRequest{
+				ExperimentID: fmt.Sprintf("%d", nonExistingExperimentID),
+			},
+			error: api.NewResourceDoesNotExistError(
+				fmt.Sprintf(
+					`unable to find experiment with id '%d': error getting experiment by id: %d: record not found`,
+					nonExistingExperimentID,
+					nonExistingExperimentID,
+				),
 			),
 		},
 	}
 	for _, tt := range tests {
 		s.T().Run(tt.name, func(T *testing.T) {
 			resp := api.ErrorResponse{}
+			client := s.MlflowClient.WithMethod(
+				http.MethodPost,
+			).WithRequest(
+				tt.request,
+			).WithResponse(
+				&resp,
+			)
+			if tt.namespace != "" {
+				client = client.WithNamespace(
+					tt.namespace,
+				)
+			}
```

Comment:
so if you remove test which I mentioned and remove `namespace: default` so in that case you don't need to have such check.